### DISCUSSION TITLE
Potential for a simple Widget?

### DISCUSSION THREAD

**Discussion by TechnoidAlpha** [Bartuzen/qBitController#113] [Ideas]
Hi, I just started using this remote app on my phone and love it, it even scales nicely on my Galaxy fold 4 (which not a lot of apps can do)

I was wondering if there was any future plans for a simple or advanced widget that can update at an interval.

Ideally it could show the total torrents downloading, seeding and paused. Also showing the current download/upload speed. This would be for the entire qbittorrent server and not for individual torrents.

**Comment by Bartuzen** [comment]
I plan to add widgets, but I can't say when I'll get to it.
